pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/admin/UserAdminService.java

```diff
@@ -995,8 +992,19 @@ protected static int deleteStudyGroup(int groupId) {
         });
     }
 
+    /**
+     * Without validating any current roles, give the user
+     * admin permission for the realm.
+     */
+    public void setGroupAdminRole(int userId, int groupId) {
+        Map<String, RoleInfo> adminRoles = getOperatorAdminRoles(userId, groupId);
+        if (adminRoles.isEmpty()) {
+            addUserRole(userId, UserAdminService.getRoleId(USER_ADMIN_ROLE), groupId);
+        }
+    }
+
 
-    @Data
+        @Data
     protected static class RoleInfo {
```

Comment:
Typo. Are you getting lint warnings for this?

Reply:
Nope, no warnings.  Begone, errant whitespace!

Reply:
I'm not sure how I setup code stye in Intellij, but my Settings | Editor | Code Style | Java is pretty complete. (Did I import a Pepper checkstyle file? I can't find any instructions.) 
I also recommend installing https://plugins.jetbrains.com/plugin/7973-sonarlint.